### DISCUSSION TITLE
Filament Page scroll up if don't put set time out

### DISCUSSION THREAD

**Discussion by Lightz2002** [filamentphp/filament#14911] [Help]
### Package

Infolist builder

### Package Version

3.0.0

### How can we help you?

![image](https://github.com/user-attachments/assets/8be58b5f-de3a-49fb-b8da-9aa265949c65)

if we don't put settimeout, it will scroll to #replies at first then scroll up to the page again.

**Comment by leandrocfe** [comment]
could you explain the problem and the goal please?
